andino_apps/README.md

```diff
@@ -0,0 +1,24 @@
+# Andino Apps
+
+This package contains more complex launh files that execute applications with the andino robot.
+
+# Usage
+
+## Gazebo classic simulation + Nav2
+
+A launch file for running the andino_gz_classic simulation and the nav2 stack is provided.
+It uses the [turtlebot3_world](https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/master) world (_Apache 2 license_) by default.
+
+```
+ ros2 launch andino_apps andino_simulation_navigation.launch.py
+```
+
+To test the navigation inside rviz:
+
+- click in 2D pose estimate button and select the initial pose of the robot
+- click in nav2 Goal button and select the final point.
+- the robot will start to move to the selected goal.
+
+You test adding obstacles inside the Gazebo simulation or use the rviz button Waypoint/ nav through Poses mode to select sequential targets.
```

Comment:
What do you mean by this? Do you want to show obstacle avoidance? If so, again a GIF will do it. However, that should come with a concrete reference to the piece of code (local planner) in charge of dealing with that.

Reply:
Maybe it would be better to just reference the Nav2 tutorials here.